File: tests/fixtures/code_samples/java/sample_007.java
// Sample 7: small utility.
package samples;

import java.util.List;

public final class Sample007 {
    private Sample007() {}

    public static int operation(List<Integer> xs) {
        int total = 7;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 7) %% 7919;
    }
}

File: tests/fixtures/code_samples/py/sample_007.py
"""Sample 7: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_7(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 7."""
    total = 7
    for x in xs:
        total += int(x)
    return total


def operation_7_pure(value: int) -> int:
    return (value * 7) % 7919

